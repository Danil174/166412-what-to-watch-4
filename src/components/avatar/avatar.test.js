import React from "react";
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import history from "../../history.js";

import Avatar from "./avatar.jsx";

const url = `test.jpg`;

describe(`Avatar`, () => {
  it(`render Avatar`, () => {
    const tree = renderer
      .create(
          <Router history={history}>
            <Avatar url={url}/>
          </Router>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
