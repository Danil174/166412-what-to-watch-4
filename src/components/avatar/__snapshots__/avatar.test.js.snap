// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar render Avatar 1`] = `
<a
  href="/favorites"
  onClick={[Function]}
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="https://htmlacademy-react-3.appspot.comtest.jpg"
      width="63"
    />
  </div>
</a>
`;
